### PR TITLE
Problem: no 'prepare' command is available

### PR DESCRIPTION
**Pull request by knekrasov**
JIRA ticket: [EOS-18909](https://jts.seagate.com/browse/EOS-18909)

According to the updated mini provisioner requirements, `hare_setup prepare --config $URL` command must be supported by our component.

Solution:
    1. Add prepare command to setup_hare (so that it is available through CLI, but effectively a no-op)
    2. Update setup.yaml to reflect the new command
    3. Fix flake8 issue in mini provisioner code.

### CODE REVIEW

**Comment by knekrasov** [comment]
@SwapnilGaonkar7 I refactored the code you had written, so it would be nice if you can review.

**Review by SwapnilGaonkar7 — APPROVED** [review]
Changes looks good.
@knekrasov Should we get 'setup.yaml' changes reviewed from privisioning team?

**Comment by knekrasov** [comment]
@SwapnilGaonkar7 As far as I know we have no process defined with respect to setup.yaml reviews. Per my understanding they will eventually review the current setup.yaml in the main branch.

@s-arya, @mssawant, @suvratjoshi Any thoughts?

File: provisioning/miniprov/hare_mp/cdf.py [hunk 1]
```diff
@@ -78,8 +78,7 @@ def _create_pool_descriptions(self) -> List[PoolDesc]:
 
             min_pool_width = data_units_count + parity_units_count \
                 + spare_units_count
-            if (data_devices_count !=
-                    0) and not (data_devices_count >= min_pool_width):
+            if data_devices_count and (data_devices_count < min_pool_width):
```
Comment (mssawant) [inline provisioning/miniprov/hare_mp/cdf.py:81]: I think it is good to have an explicit check for integers, `if data_devices_count > 0` in this case. It helps to clearly differentiate a boolean from an integer (I think it is a good practice even though a variable name has `count` in it).

**Review by mssawant — APPROVED** [review]
Minor comment, no more comments.

File: provisioning/miniprov/hare_mp/cdf.py (hunk 1 above)
Comment (mssawant) [reply provisioning/miniprov/hare_mp/cdf.py:81]: Merging the patch.

**Pull request merged**